### DISCUSSION TITLE
adda bookmark emulatar

### DISCUSSION THREAD

**Discussion by FoxMoss** [3kh0/ext-remover#641] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
i think 3kh0 should make a bookmark emulator so that my school cant blaock the meat one :((( :heart:

**Comment by jessiejs** [comment]
yaes, this is vrye gdod i dea.

**Comment by Aka-but-nice** [comment]
> i think 3kh0 should make a bookmark emulator so that my school cant blaock the meat one :((( ❤️

There's no way! I can't find a no option! Aww.

**Comment by bartholemuew** [comment]
whats an emulatar????

**Reply by TheGooferFox** [reply]
An "emulatar" is like a simulation of something like running an emulator via gameboy and playing it on your computer or smth like bluestacks that runs a custom android system blah blah

**Comment by TheGooferFox** [comment]
Lemme clear this question up rq
Y o u c a n t
if you cant fucking run javascript ur fucked dude we cant help my school did the same find new bypasses or websites lol or do ur work its simple
